src/pages/ProfilePage.jsx

```diff
@@ -226,11 +226,13 @@ function ProfilePage(props) {
                                 </div>
                             </div>
                         </div>
-                        <div className="row flex-grow-1 justify-center justify-content-center mx-0">
-                            <div className="col-6 col-lg-4 pt-4">
-                                <TextButton className="px-5" hsl={[97, 43, 70]} text="Editar" onClick={() => navigate(`manage`)} />
+                        {(user.role === 'PUBLISHER' || user.role === 'COORDINATOR' || user.role === 'ADMIN') && (
```

Comment:
```suggestion
                        {user.id !== 1 && (
```

- Levando em consideração que os demais usuários poderão atualizar seu próprio perfil pelo app, acredito que assim faça mais sentido. Mas vale discutir..
- No contexto dessa branch o app não tem rota pra página de editar usuário, então se formos manter a ideia citada acima, em que usuários que não sejam Visitante possam atualizar o próprio perfil, o ideal seria adicionar a rota para tal.